cmd/daemon/kubernetes/kubernetes.go

```diff
@@ -27,7 +29,15 @@ var (
 )
 
 func NewClient() (*Client, error) {
```

Comment:
What do you think about moving the environment lookup out to a flag with a default value in `cmd/daemon/command/start.go` instead? Just to avoid having environment lookups in the code.

Reply:
I love it. I just wanted to keep changes to an absolute minimum. Can add that change though. 👍